go.mod

```diff
@@ -1,32 +1,16 @@
 module github.com/nukosuke/terraform-provider-zendesk
 
 require (
-	github.com/agext/levenshtein v1.2.2 // indirect
-	github.com/apparentlymart/go-cidr v1.0.0 // indirect
-	github.com/armon/go-radix v1.0.0 // indirect
-	github.com/aws/aws-sdk-go v1.19.46 // indirect
-	github.com/blang/semver v0.0.0-20190414102917-ba2c2ddd8906 // indirect
 	github.com/golang/mock v1.3.1
-	github.com/hashicorp/go-getter v1.3.0 // indirect
-	github.com/hashicorp/go-hclog v0.9.2 // indirect
-	github.com/hashicorp/go-plugin v1.0.0 // indirect
 	github.com/hashicorp/hcl v1.0.0 // indirect
-	github.com/hashicorp/hcl2 v0.0.0-20190609194556-318e80eefe28 // indirect
 	github.com/hashicorp/hil v0.0.0-20190212132231-97b3a9cdfa93 // indirect
-	github.com/hashicorp/terraform v0.11.14
-	github.com/hashicorp/yamux v0.0.0-20181012175058-2f1d1f20f75d // indirect
-	github.com/mattn/go-isatty v0.0.8 // indirect
-	github.com/mitchellh/cli v1.0.0 // indirect
-	github.com/mitchellh/copystructure v1.0.0 // indirect
-	github.com/mitchellh/go-homedir v1.1.0 // indirect
-	github.com/mitchellh/go-wordwrap v1.0.0 // indirect
-	github.com/mitchellh/hashstructure v1.0.0 // indirect
+	github.com/hashicorp/terraform v0.12.11
+	github.com/hashicorp/terraform-plugin-sdk v1.2.0
```

Comment:
It's fine here, but maybe we should replace all "github.com/hashicorp/terraform" to "github.com/hashicorp/terraform-plugin-sdk" in future.

https://www.terraform.io/docs/extend/plugin-sdk.html

> As of September 2019, Terraform provider developers importing the Go module github.com/hashicorp/terraform, known as Terraform Core, should switch to github.com/hashicorp/terraform-plugin-sdk, the Terraform Plugin SDK, instead.

Reply:
+1 - I'll wrap that up in a separate PR.

Reply:
See #169